WCSim.mac

```diff
@@ -52,7 +53,7 @@
 #/WCSim/LCConfig Mirror_OldLC # Mirror design in a branch of archive/LightCone at 2011.
 #/WCSim/LCConfig Mirror_2018Oct # A short mirror design.
 
-#/WCSim/Construct # This must be uncommented in order to construct a new detector configuration.
```

Comment:
Please put back all the defaults here in WCSim.mac and just add in the new part you need.

Reply:
Ok, I did it.